pages/thc/[processRef]/review.tsx

```diff
@@ -162,46 +167,59 @@ const ReviewPage: NextPage = () => {
 
           setSignatures((sigs) => ({ ...sigs, [selectedTenantId]: value }));
         }}
+        readOnly={isManager}
       />
```

Comment:
Note that `Signature` isn't actually populated from the database (intentionally - see the comment by the `useState` call). We probably want to show images instead of the component for the manager.

Reply:
This actually crept in, I removed it in this PR because I'm going to handle it in the next one, but this is good to know for the future.